src/windows/views/properties_tableview.py

```diff
@@ -545,6 +534,16 @@ def contextMenuEvent(self, event):
 
             # Add menu options for keyframes
             menu = QMenu(self)
+
+            if self.property_type == "font":
+                font_action = menu.addAction(_("Select font"))
```

Comment:
```suggestion
                font_action = menu.addAction(_("Change font"))
```
I noticed we already have this string in the translations.